docusaurus/docs/iOS/06-advanced/06-apply-video-filters.mdx

```diff
@@ -292,99 +292,86 @@ Button {
 
 ## Audio Filters
 
-The StreamVideo SDK also supports custom audio processing of the local track. This opens up possibilities for custom echo filtering, voice changing or other audio effects.
+The StreamVideo SDK also supports audio processing of the local track. This opens up possibilities for noise cancellation, voice changing or other audio effects.
 
-If you want to have custom audio processing, you need to provide your own implementation of the `AudioProcessingModule` protocol in our `VideoConfig`. 
+If you want to have audio processing, you need to either provide your own implementation of the `AudioProcessingModule` protocol in our `VideoConfig` or use the provided one implemented on `StreamAudioFilterProcessingModule`. 
```

Comment:
I think the focus should no longer be on the robot voice filter. We should first expose a new section "Noise cancellation" here, explain how to turn it on with the code sample, and then proceed with the robot voice filter.

Reply:
That will happen once we have the feature complete (the guideline is to not have anything referencing NC before licensing, right?)

Reply:
nothing to see here haha